content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,330 @@
+    {{< /text >}}
+
+    When a valid JWT is attached, it returns the HTTP code 200.
+
+1.  Verify that the connection to the `httpbin` service fails when the JWT is not attached:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    {{< /text >}}
+
+    When no valid JWT is attached, it returns the HTTP code 401.
+
+## Configure groups-based authorization
+
+This section creates a policy to authorize the access to the `httpbin`
+service if the requests are originated from specific groups.
+As there may be some delays due to caching and other propagation overhead,
+wait until the newly defined RBAC policy to take effect.
+
+1.  Enable the Istio RBAC for the namespace:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["rbac-groups-test-ns"]
+    EOF
+    {{< /text >}}
+
+1.  Once the RBAC policy takes effect, verify that Istio rejected the curl
+connection to the `httpbin` service:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    Once the RBAC policy takes effect, the command returns the HTTP code 403.
+
+1.  To give read access to the `httpbin` service, create the `httpbin-viewer`
+service role:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRole
+    metadata:
+      name: httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      rules:
+      - services: ["httpbin.rbac-groups-test-ns.svc.cluster.local"]
+        methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+1.  To assign the `httpbin-viewer` role to users in `group1`, create the
+`bind-httpbin-viewer` service role binding.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRoleBinding
+    metadata:
+      name: bind-httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      subjects:
+      - properties:
+          request.auth.claims[groups]: "group1"
+      roleRef:
+        kind: ServiceRole
+        name: "httpbin-viewer"
+    EOF
+    {{< /text >}}
+
+    Alternatively, you may also specify the group under `subjects.group`
+    as follows. The two ways for specifying the group are equivalent.
+    Currently `request.auth.claims` and `subjects.group` are the only properties
+    that support matching against a list of strings in the JWT token.
```

Comment:
Alternatively, you may specify `group` field under `subjects` directly. The two ways for specifying the group are equivalent.

Do not use `subjects.group`. This is not an attribute like `request.auth.claims`.

Reply:
Done.

Reply:
The same comments apply to "Currently ...". Use `group` field under `subjects` instead of `subjects.group`.

Reply:
Done.

Reply:
Avoid using conjunctive: may, might, could, would. I question why you are even mentioning this here. Do we expect people to do this directly? 
Unless you consider this method crucial, I would leave the text out. There are multiple paths to do stuff, but I always recommend to document the easiest and cleanest one for readers. Giving multiple alternatives, when not really necessary, tends to create confusion.

Reply:
This method is included because according to https://github.com/istio/istio/blob/master/vendor/istio.io/api/rbac/v1alpha1/rbac.proto#L177, the policy maker may directly define group under subject.

Reply:
If you think it is needed, I recommend this fix:
Alternatively, you can specify the group under `subjects.group` Both ways to specify the group are equivalent. Currently, Istio only supports matching against a list of strings in the JWT for the `request.auth.claims` and the `subjects.group` properties.

Reply:
I forgot to add the final sentence introducing the command:
To specify the group under subjects.group, use the following command:

Reply:
Done.